waku/v2/node/waku_node.nim

```diff
@@ -148,6 +148,13 @@ proc new*(T: type WakuNode,
           nameResolver: NameResolver = nil,
           sendSignedPeerRecord = false,
           dns4DomainName = none(string),
+          discv5Enabled = false,
+          discv5TableIpLimit = none(uint),
```

Comment:
perhaps 7 fields for discv5 is too much? WakuNode.new() is starting to have too many parameters.

regarding discv5TableIpLimit, discv5BucketIpLimit, discv5BitsPerHop not sure if having these configurable is too much. default values should be fine, and whatever is not default should be carefully studied. but out of scope now i guess.


Using `DiscoveryConfig` can perhaps reduce the number of fields there? and if empty defaults to `defaultDiscoveryConfig`
```
  DiscoveryConfig(
    tableIpLimits: TableIpLimits(
      tableIpLimit: tableIpLimit,
      bucketIpLimit: bucketIpLimit),
    bitsPerHop: bitsPerHop
```

Reply:
Addressed in [185a10a](https://github.com/waku-org/nwaku/pull/1512/commits/185a10ad6637c8c13f2e58c5d7305d290c6a3982), thanks!

Reply:
Indeed, this is one of the reasons for not having a discv5 within the waku node. You configure discv5 outside, so you don't have to thread down any configuration.